docs/conf.py

```diff
@@ -10,9 +10,9 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
I wonder why we need this? It's possible that if we move the `watch.py` into `docs` this might go away?

Reply:
This is a relic of my attempt to enable `autodoc` before I switched to the Bazel approach. It's not needed anymore, thanks for catching this